frontend/components/Card.tsx

```diff
@@ -0,0 +1,63 @@
+import { Card as AntdCard } from 'antd';
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+import { useRouter } from 'next/router';
+import SvgRightArrow from '../public/right-arrow.svg';
+
+const StyledCard = styled(AntdCard)`
```

Comment:
Seeing some weird behavior when I extend the screen a little bit. The right arrow gets hidden from view as the screen is widened, don't know if its something goin g on with the antd component, but definitely take a look at it please

<img width="705" alt="Screen Shot 2021-11-07 at 5 36 10 PM" src="https://user-images.githubusercontent.com/17476012/140664418-6b8e625d-96d0-40f7-80ad-325915c487fc.png">

Reply:
![image](https://user-images.githubusercontent.com/51216746/140665200-4ba8d461-da7a-42c5-9f8e-4a3bf47efcad.png)

I'm not sure why that happens on your screen. I'm wondering if it's because you are using safari (does this also happen when you try it on chrome?). Also, do you have the most updated version of this branch?

Reply:
Hmmm very weird, you're right. It doesn't happen on Chrome, but it happens on safari

Reply:
I was able to replicate part of the issue in Safari. The arrows are for some reason left-aligned in Safari, but they aren't being cut off by the card on my screen.